### ISSUE TITLE
Unable to install rust HEAD with Homebrew

### ISSUE THREAD

**Issue by blixt** [rust-lang/rust#31490]
I've been trying to install Rust with `brew install --HEAD` but it's not working, so I figured I'd report it here. The error appears to be in the configure step (git error about "test-installer-v2").

```
$ brew reinstall rust --HEAD -v
==> Reinstalling rust
rm /usr/local/bin/cargo
rm /usr/local/bin/rust-lldb
rm /usr/local/bin/rustc
rm /usr/local/bin/rustdoc
rm /usr/local/etc/bash_completion.d/cargo
rm /usr/local/lib/libarena-ca9f0d77.dylib
rm /usr/local/lib/libflate-ca9f0d77.dylib
rm /usr/local/lib/libfmt_macros-ca9f0d77.dylib
rm /usr/local/lib/libgetopts-ca9f0d77.dylib
rm /usr/local/lib/libgraphviz-ca9f0d77.dylib
rm /usr/local/lib/liblog-ca9f0d77.dylib
rm /usr/local/lib/librbml-ca9f0d77.dylib
rm /usr/local/lib/librustc-ca9f0d77.dylib
rm /usr/local/lib/librustc_back-ca9f0d77.dylib
rm /usr/local/lib/librustc_borrowck-ca9f0d77.dylib
rm /usr/local/lib/librustc_data_structures-ca9f0d77.dylib
rm /usr/local/lib/librustc_driver-ca9f0d77.dylib
rm /usr/local/lib/librustc_front-ca9f0d77.dylib
rm /usr/local/lib/librustc_lint-ca9f0d77.dylib
rm /usr/local/lib/librustc_llvm-ca9f0d77.dylib
rm /usr/local/lib/librustc_metadata-ca9f0d77.dylib
rm /usr/local/lib/librustc_mir-ca9f0d77.dylib
rm /usr/local/lib/librustc_platform_intrinsics-ca9f0d77.dylib
rm /usr/local/lib/librustc_plugin-ca9f0d77.dylib
rm /usr/local/lib/librustc_privacy-ca9f0d77.dylib
rm /usr/local/lib/librustc_resolve-ca9f0d77.dylib
rm /usr/local/lib/librustc_trans-ca9f0d77.dylib
rm /usr/local/lib/librustc_typeck-ca9f0d77.dylib
rm /usr/local/lib/librustdoc-ca9f0d77.dylib
rm /usr/local/lib/libserialize-ca9f0d77.dylib
rm /usr/local/lib/libstd-ca9f0d77.dylib
rm /usr/local/lib/libsyntax-ca9f0d77.dylib
rm /usr/local/lib/libterm-ca9f0d77.dylib
rm /usr/local/lib/libtest-ca9f0d77.dylib
rm /usr/local/lib/rustlib
rm /usr/local/share/doc/cargo
rm /usr/local/share/doc/rust
rm /usr/local/share/man/man1/cargo.1
rm /usr/local/share/man/man1/rustc.1
rm /usr/local/share/man/man1/rustdoc.1
rm /usr/local/share/zsh/site-functions/_cargo
==> Cloning https://github.com/rust-lang/rust.git
Updating /Library/Caches/Homebrew/rust--git
git config remote.origin.url https://github.com/rust-lang/rust.git
git config remote.origin.fetch +refs/heads/master:refs/remotes/origin/master
git fetch origin
==> Checking out branch master
git checkout -f master --
Already on 'master'
Your branch is up-to-date with 'origin/master'.
git reset --hard origin/master
HEAD is now at e06f692 Auto merge of #31468 - pitdicker:fs_tests_cleanup, r=alexcrichton
git submodule foreach --recursive git submodule sync
Entering 'src/compiler-rt'
Entering 'src/jemalloc'
Entering 'src/liblibc'
Entering 'src/llvm'
Entering 'src/rt/hoedown'
Entering 'src/rust-installer'
Synchronizing submodule url for 'test/rust-installer-v1'
Synchronizing submodule url for 'test/rust-installer-v2'
Entering 'src/rust-installer/test/rust-installer-v1'
Entering 'src/rust-installer/test/rust-installer-v2'
Synchronizing submodule url for 'test/rust-installer-v1'
Entering 'src/rust-installer/test/rust-installer-v2/test/rust-installer-v1'
git submodule update --init --recursive
==> ./configure --prefix=/usr/local/Cellar/rust/HEAD --disable-rpath --enable-clang --release-channel=nightly
configure: looking for configure programs
configure: found program 'cmp'
configure: found program 'mkdir'
configure: found program 'printf'
configure: found program 'cut'
configure: found program 'head'
configure: found program 'grep'
configure: found program 'xargs'
configure: found program 'cp'
configure: found program 'find'
configure: found program 'uname'
configure: found program 'date'
configure: found program 'tr'
configure: found program 'sed'
configure: found program 'file'
configure: found program 'make'
configure: inspecting environment
configure: recreating config.tmp
configure:
configure: processing ./configure args
configure:
configure: CFG_DISABLE_RPATH    := 1
configure: CFG_LOCALSTATEDIR    := /var/lib
configure: CFG_SYSCONFDIR       := /etc
configure: CFG_DATADIR          := /share
configure: CFG_INFODIR          := /share/info
configure: CFG_LLVM_ROOT        :=
configure: CFG_PYTHON           :=
configure: CFG_JEMALLOC_ROOT    :=
configure: CFG_BUILD            := x86_64-apple-darwin
configure: CFG_ANDROID_CROSS_PATH := /opt/ndk_standalone
configure: CFG_I686_LINUX_ANDROID_NDK :=
configure: CFG_ARM_LINUX_ANDROIDEABI_NDK :=
configure: CFG_AARCH64_LINUX_ANDROID_NDK :=
configure: CFG_NACL_CROSS_PATH  :=
configure: CFG_RELEASE_CHANNEL  := nightly
configure: CFG_MUSL_ROOT        := /usr/local
configure: CFG_EXTRA_FILENAME   :=
configure: CFG_DEFAULT_LINKER   := cc
configure: CFG_DEFAULT_AR       := ar
configure: CFG_LIBDIR           := /usr/local/Cellar/rust/HEAD/lib
configure:
configure: validating ./configure args
configure:
configure: overriding settings for nightly
configure: CFG_ENABLE_LLVM_ASSERTIONS := 1
configure: CFG_BOOTSTRAP_KEY    := 11:59:03
configure:
configure: looking for build programs
configure:
configure: CFG_CURLORWGET       := /usr/bin/curl (7.43.0)
configure: CFG_PYTHON           := /usr/bin/python2.7
configure: CFG_GIT              := /usr/local/Library/ENV/4.3/git (2.7.1)
configure: CFG_MD5              := /sbin/md5
configure: CFG_MD5SUM           :=
configure: CFG_HASH_COMMAND     := /sbin/md5 -q | cut -c 1-8
configure: CFG_CLANG            := /usr/local/Library/ENV/4.3/clang++ (7.0.2)
configure: CFG_CCACHE           :=
configure: CFG_GCC              := /usr/local/Library/ENV/4.3/gcc (7.0.2)
configure: CFG_LD               := /usr/local/Library/ENV/4.3/ld
configure: CFG_VALGRIND         :=
configure: CFG_PERF             :=
configure: CFG_ISCC             :=
configure: CFG_ANTLR4           :=
configure: CFG_GRUN             :=
configure: CFG_FLEX             := /usr/bin/flex (2.5.35)
configure: CFG_BISON            := /usr/bin/bison (2.3)
configure: CFG_GDB              :=
configure: CFG_LLDB             := /usr/bin/lldb (340.4.119.1)
configure: CFG_DISABLE_VALGRIND_RPASS := 1
configure: CFG_LLDB_VERSION     := lldb-340.4.119.1
configure: CFG_LLDB_PYTHON_DIR  := /Applications/Xcode.app/Contents/Sh ...
configure:
configure: looking for target specific programs
configure:
configure: CFG_ADB              :=
configure: CFG_ENABLE_CLANG     := 1
configure: skipping compiler inference steps; using provided CC=clang
configure:
configure: note, user-provided CC looks like clang; CC=clang.
configure:
configure: CFG_USING_CLANG      := 1
configure:
configure: found ok version of APPLE CLANG: 7.0.2
configure:
configure: CFG_CC               := clang
configure: CFG_CXX              := clang++
configure: CFG_STDCPP_NAME      := stdc++
configure:
configure: making directories
configure:
configure: mkdir -p doc
configure: mkdir -p doc/std
configure: mkdir -p doc/extra
configure: mkdir -p dl
configure: mkdir -p tmp
configure: mkdir -p dist
configure: mkdir -p x86_64-apple-darwin/llvm
configure: mkdir -p x86_64-apple-darwin/rustllvm
configure: mkdir -p x86_64-apple-darwin/rt
configure: mkdir -p x86_64-apple-darwin/rt/stage0
configure: mkdir -p x86_64-apple-darwin/rt/jemalloc
configure: mkdir -p x86_64-apple-darwin/rt/compiler-rt
configure: mkdir -p x86_64-apple-darwin/rt/stage0/isaac
configure: mkdir -p x86_64-apple-darwin/rt/stage0/sync
configure: mkdir -p x86_64-apple-darwin/rt/stage0/test
configure: mkdir -p x86_64-apple-darwin/rt/stage0/arch/i386
configure: mkdir -p x86_64-apple-darwin/rt/stage0/arch/x86_64
configure: mkdir -p x86_64-apple-darwin/rt/stage0/arch/arm
configure: mkdir -p x86_64-apple-darwin/rt/stage0/arch/aarch64
configure: mkdir -p x86_64-apple-darwin/rt/stage0/arch/mips
configure: mkdir -p x86_64-apple-darwin/rt/stage0/arch/powerpc
configure: mkdir -p x86_64-apple-darwin/rt/stage1
configure: mkdir -p x86_64-apple-darwin/rt/stage1/isaac
configure: mkdir -p x86_64-apple-darwin/rt/stage1/sync
configure: mkdir -p x86_64-apple-darwin/rt/stage1/test
configure: mkdir -p x86_64-apple-darwin/rt/stage1/arch/i386
configure: mkdir -p x86_64-apple-darwin/rt/stage1/arch/x86_64
configure: mkdir -p x86_64-apple-darwin/rt/stage1/arch/arm
configure: mkdir -p x86_64-apple-darwin/rt/stage1/arch/aarch64
configure: mkdir -p x86_64-apple-darwin/rt/stage1/arch/mips
configure: mkdir -p x86_64-apple-darwin/rt/stage1/arch/powerpc
configure: mkdir -p x86_64-apple-darwin/rt/stage2
configure: mkdir -p x86_64-apple-darwin/rt/stage2/isaac
configure: mkdir -p x86_64-apple-darwin/rt/stage2/sync
configure: mkdir -p x86_64-apple-darwin/rt/stage2/test
configure: mkdir -p x86_64-apple-darwin/rt/stage2/arch/i386
configure: mkdir -p x86_64-apple-darwin/rt/stage2/arch/x86_64
configure: mkdir -p x86_64-apple-darwin/rt/stage2/arch/arm
configure: mkdir -p x86_64-apple-darwin/rt/stage2/arch/aarch64
configure: mkdir -p x86_64-apple-darwin/rt/stage2/arch/mips
configure: mkdir -p x86_64-apple-darwin/rt/stage2/arch/powerpc
configure: mkdir -p x86_64-apple-darwin/rt/stage3
configure: mkdir -p x86_64-apple-darwin/rt/stage3/isaac
configure: mkdir -p x86_64-apple-darwin/rt/stage3/sync
configure: mkdir -p x86_64-apple-darwin/rt/stage3/test
configure: mkdir -p x86_64-apple-darwin/rt/stage3/arch/i386
configure: mkdir -p x86_64-apple-darwin/rt/stage3/arch/x86_64
configure: mkdir -p x86_64-apple-darwin/rt/stage3/arch/arm
configure: mkdir -p x86_64-apple-darwin/rt/stage3/arch/aarch64
configure: mkdir -p x86_64-apple-darwin/rt/stage3/arch/mips
configure: mkdir -p x86_64-apple-darwin/rt/stage3/arch/powerpc
configure: mkdir -p x86_64-apple-darwin/stage0/bin
configure: mkdir -p x86_64-apple-darwin/stage0/lib
configure: mkdir -p x86_64-apple-darwin/stage0/test
configure: mkdir -p x86_64-apple-darwin/stage0/lib/rustlib/x86_64-apple-darwin/bin
configure: mkdir -p x86_64-apple-darwin/stage0/lib/rustlib/x86_64-apple-darwin/lib
configure: mkdir -p x86_64-apple-darwin/stage1/bin
configure: mkdir -p x86_64-apple-darwin/stage1/lib
configure: mkdir -p x86_64-apple-darwin/stage1/test
configure: mkdir -p x86_64-apple-darwin/stage1/lib/rustlib/x86_64-apple-darwin/bin
configure: mkdir -p x86_64-apple-darwin/stage1/lib/rustlib/x86_64-apple-darwin/lib
configure: mkdir -p x86_64-apple-darwin/stage2/bin
configure: mkdir -p x86_64-apple-darwin/stage2/lib
configure: mkdir -p x86_64-apple-darwin/stage2/test
configure: mkdir -p x86_64-apple-darwin/stage2/lib/rustlib/x86_64-apple-darwin/bin
configure: mkdir -p x86_64-apple-darwin/stage2/lib/rustlib/x86_64-apple-darwin/lib
configure: mkdir -p x86_64-apple-darwin/stage3/bin
configure: mkdir -p x86_64-apple-darwin/stage3/lib
configure: mkdir -p x86_64-apple-darwin/stage3/test
configure: mkdir -p x86_64-apple-darwin/stage3/lib/rustlib/x86_64-apple-darwin/bin
configure: mkdir -p x86_64-apple-darwin/stage3/lib/rustlib/x86_64-apple-darwin/lib
configure: mkdir -p x86_64-apple-darwin/test/run-pass
configure: mkdir -p x86_64-apple-darwin/test/run-pass-valgrind
configure: mkdir -p x86_64-apple-darwin/test/run-pass-fulldeps
configure: mkdir -p x86_64-apple-darwin/test/run-fail
configure: mkdir -p x86_64-apple-darwin/test/run-fail-fulldeps
configure: mkdir -p x86_64-apple-darwin/test/compile-fail
configure: mkdir -p x86_64-apple-darwin/test/parse-fail
configure: mkdir -p x86_64-apple-darwin/test/compile-fail-fulldeps
configure: mkdir -p x86_64-apple-darwin/test/bench
configure: mkdir -p x86_64-apple-darwin/test/perf
configure: mkdir -p x86_64-apple-darwin/test/pretty
configure: mkdir -p x86_64-apple-darwin/test/debuginfo-gdb
configure: mkdir -p x86_64-apple-darwin/test/debuginfo-lldb
configure: mkdir -p x86_64-apple-darwin/test/codegen
configure: mkdir -p x86_64-apple-darwin/test/codegen-units
configure: mkdir -p x86_64-apple-darwin/test/rustdoc
configure:
configure: configuring submodules
configure:
configure: git: submodule sync
Synchronizing submodule url for 'src/compiler-rt'
Synchronizing submodule url for 'src/jemalloc'
Synchronizing submodule url for 'src/liblibc'
Synchronizing submodule url for 'src/llvm'
Synchronizing submodule url for 'src/rt/hoedown'
Synchronizing submodule url for 'src/rust-installer'
configure: git: submodule init
configure: git: submodule update
configure: git: submodule foreach sync
Entering 'src/compiler-rt'
Entering 'src/jemalloc'
Entering 'src/liblibc'
Entering 'src/llvm'
Entering 'src/rt/hoedown'
Entering 'src/rust-installer'
Synchronizing submodule url for 'test/rust-installer-v1'
Synchronizing submodule url for 'test/rust-installer-v2'
Entering 'src/rust-installer/test/rust-installer-v1'
Entering 'src/rust-installer/test/rust-installer-v2'
fatal: /usr/local/Cellar/git/2.7.1/libexec/git-core/git-submodule cannot be used without a working tree.
Stopping at 'src/rust-installer/test/rust-installer-v2'; script returned non-zero status.
Stopping at 'src/rust-installer'; script returned non-zero status.
configure: error: git failed
ln -s ../../Cellar/rust/1.6.0/etc/bash_completion.d/cargo cargo
ln -s ../Cellar/rust/1.6.0/bin/cargo cargo
ln -s ../Cellar/rust/1.6.0/bin/rust-lldb rust-lldb
ln -s ../Cellar/rust/1.6.0/bin/rustc rustc
ln -s ../Cellar/rust/1.6.0/bin/rustdoc rustdoc
ln -s ../../Cellar/rust/1.6.0/share/doc/cargo cargo
ln -s ../../Cellar/rust/1.6.0/share/doc/rust rust
ln -s ../../../Cellar/rust/1.6.0/share/man/man1/cargo.1 cargo.1
ln -s ../../../Cellar/rust/1.6.0/share/man/man1/rustc.1 rustc.1
ln -s ../../../Cellar/rust/1.6.0/share/man/man1/rustdoc.1 rustdoc.1
ln -s ../../../Cellar/rust/1.6.0/share/zsh/site-functions/_cargo _cargo
ln -s ../Cellar/rust/1.6.0/lib/libarena-ca9f0d77.dylib libarena-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/libflate-ca9f0d77.dylib libflate-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/libfmt_macros-ca9f0d77.dylib libfmt_macros-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/libgetopts-ca9f0d77.dylib libgetopts-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/libgraphviz-ca9f0d77.dylib libgraphviz-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/liblog-ca9f0d77.dylib liblog-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librbml-ca9f0d77.dylib librbml-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc-ca9f0d77.dylib librustc-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_back-ca9f0d77.dylib librustc_back-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_borrowck-ca9f0d77.dylib librustc_borrowck-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_data_structures-ca9f0d77.dylib librustc_data_structures-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_driver-ca9f0d77.dylib librustc_driver-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_front-ca9f0d77.dylib librustc_front-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_lint-ca9f0d77.dylib librustc_lint-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_llvm-ca9f0d77.dylib librustc_llvm-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_metadata-ca9f0d77.dylib librustc_metadata-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_mir-ca9f0d77.dylib librustc_mir-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_platform_intrinsics-ca9f0d77.dylib librustc_platform_intrinsics-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_plugin-ca9f0d77.dylib librustc_plugin-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_privacy-ca9f0d77.dylib librustc_privacy-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_resolve-ca9f0d77.dylib librustc_resolve-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_trans-ca9f0d77.dylib librustc_trans-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustc_typeck-ca9f0d77.dylib librustc_typeck-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/librustdoc-ca9f0d77.dylib librustdoc-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/libserialize-ca9f0d77.dylib libserialize-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/libstd-ca9f0d77.dylib libstd-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/libsyntax-ca9f0d77.dylib libsyntax-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/libterm-ca9f0d77.dylib libterm-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/libtest-ca9f0d77.dylib libtest-ca9f0d77.dylib
ln -s ../Cellar/rust/1.6.0/lib/rustlib rustlib
==> Formula
Path: /usr/local/Library/Formula/rust.rb
==> Configuration
HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/Homebrew/homebrew
HEAD: 0891aa993cd7417dff15a2ad7c831a615727b5ec
Last commit: 2 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: 8-core 64-bit haswell
OS X: 10.11.2-x86_64
Xcode: 7.2.1
CLT: 7.2.0.0.1.1447826929
Clang: 7.0 build 700
X11: 2.7.8 => /opt/X11
System Ruby: 2.0.0-p645
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.8.0_60, 1.8.0_45, 1.7.0_71
==> ENV
HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
MAKEFLAGS: -j8
CMAKE_PREFIX_PATH: /usr/local/opt/openssl:/usr/local
CMAKE_INCLUDE_PATH: /usr/include/libxml2:/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
PKG_CONFIG_PATH: /usr/local/opt/openssl/lib/pkgconfig:/usr/local/opt/libssh2/lib/pkgconfig
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Library/ENV/pkgconfig/10.11
ACLOCAL_PATH: /usr/local/share/aclocal
PATH: /usr/local/Library/ENV/4.3:/usr/local/opt/cmake/bin:/usr/local/opt/pkg-config/bin:/usr/local/opt/openssl/bin:/usr/local/opt/git/bin:/usr/bin:/bin:/usr/sbin:/sbin

Error: rust HEAD did not build
Logs:
     /Users/blixt/Library/Logs/Homebrew/rust/01.configure
     /Users/blixt/Library/Logs/Homebrew/rust/01.configure.cc

These open issues may also help:
Option to keep Rust src for racer https://github.com/Homebrew/homebrew/pull/48377
```

**Comment by blixt** [comment]
Installing rust manually from source (`./configure && make && make install`) worked, so this seems to be an issue with how Homebrew behaves. I just can't tell if it's a bug with how the rust formula is set up, or with Homebrew itself.

**Comment by steveklabnik** [comment]
Since it builds locally, yeah, this would seem to be a homebrew issue. I don't see anything obviously wrong in https://github.com/Homebrew/homebrew/blob/master/Library/Formula/rust.rb , but since it does build for you without homebrew, I'm going to give this a close. You should file an issue with Homebrew. Thanks!

**Comment by palad1** [comment]
I cannot build 1.6.0 from source, the snapshots.txt file is referencing an old cargo that breaks with the current crates.io :/

**Comment by steveklabnik** [comment]
@palad1 that seems bad, please file a bug against homebrew!

**Comment by palad1** [comment]
Thanks, just so I'm clear, I can use cargo HEAD  with rust 1.6.0?

**Comment by steveklabnik** [comment]
I _think_ so, but we only make strong guarantees about the version that ships with Rust itself. Rust 1.6.0 shipped with `cargo 0.7.0-nightly (1af03be 2015-12-08)`.
